tourjs-react/src/tourjs-client-shared/drawing3d.ts

```diff
@@ -678,6 +678,38 @@ function buildRoad(raceState:RaceState):THREE.Mesh[] {
   return [roadMesh, farGrassMesh, skyMesh, nearGrassMesh, spaceMesh]
 }
 
+
+function buildFinishLine(map:RideMap):THREE.Mesh {
+  const finishLineLocation = map.getLength();
+  const length = 5;
+  const height = 0.01;
+  const width = Planes.RoadNear - Planes.RoadFar
+
+  
+  const finishLineTexture = new THREE.TextureLoader().load( "/finishline.jpg" );
+  finishLineTexture.wrapS = THREE.RepeatWrapping;
+  finishLineTexture.wrapT = THREE.RepeatWrapping;
+  finishLineTexture.repeat.set(0.45,3);         // These values get aspect ratio of checkerboard right (trial and error)
```

Comment:
I bet 3/0.45 is the same as the width/height of your jpg

Reply:
Indeed it is! Every once in awhile the universe surprises me with it's elegancy!